fix 'defied' to 'defined' (#2105)
fishtown-analytics/docs.getdbt.com: website/docs/reference/configs-and-properties.md

### Config inheritance

> Configurations are prioritized in order of specificity, which is generally the order above: an in-file `config()` block takes precedence over properties defined in a `.yml` file, which takes precedence over a config defined in the project file. (Note that generic tests work a little differently when it comes to specificity. See [test configs](test-configs).)

Reviewer: ```suggestion
dbt prioritizes configurations in order of specificity, from most specificity to least specificity. This generally follows the order above: an in-file `config()` block --> properties defined in a `.yml` file --> config defined in the project file. 

Note - Generic tests work a little differently when it comes to specificity. See [test configs](test-configs).
```
